data/xml/schema.rnc

```diff
@@ -43,7 +43,8 @@ Paper = element paper {
    & element url { xsd:anyURI }?
    & element video {
        attribute href { xsd:anyURI },
-       attribute tag { text }
+       attribute tag { text },
+       attribute permission { text }?
```

Comment:
Unless you envision more complicated cases to arise in the future, I would suggest making this `xsd:boolean` and changing the yes/no to true/false, respectively.